packages/eas-cli/src/commands/build/index.ts

```diff
@@ -27,10 +27,6 @@ export default class Build extends Command {
 
   static flags = {
     platform: flags.enum({ char: 'p', options: ['android', 'ios', 'all'] }),
-    'skip-credentials-check': flags.boolean({
```

Comment:
maybe it would be better to leave that flag here, mark it as deprecated and throw a better error informing that flag is not necessary anymore